interfaces/Portalicious/src/app/pages/users/add-user-form/add-user-form.component.html

```diff
@@ -0,0 +1,34 @@
+<app-form-sidebar
+  [(visible)]="formVisible"
+  [formGroup]="formGroup"
+  [mutation]="userMutation"
+  formTitle="Add user"
+  i18n-formTitle
+>
+  <app-form-field-wrapper
+    label="User name"
+    i18n-label
+    [errorMessage]="formFieldErrors()('displayNameValue')"
+  >
+    <input
+      pInputText
+      type="text"
+      formControlName="usernameValue"
+      autocomplete="off"
+      class="font-mono"
```

Comment:
```suggestion
```

Sorry I didn't notice this before. I don't think this should be mono (according to the designs at least).